rhcos-continuous.repo

```diff
@@ -1,5 +1,5 @@
 [rhcos-continuous]
-name=Bleeding edge of packages relevant to RHCOS
-baseurl=https://s3.amazonaws.com/aos-ci/rhcos/rdgo
+name=Packages built by this pipeline
+baseurl=http://aos-ostree.rhev-ci-vms.eng.rdu2.redhat.com/rhcos/rdgo/build/
```

Comment:
@cgwalters How/where is baseurl used?  I'm trying to wrap my head around the build pipeline :)

Reply:
This `.repo` file is used as part of the treecompose pipeline.  It provides the RPMs built by the `rdgo` pipeline.

It was changed to this internal location because the S3 bucket where the RPMs were originally getting synced to was private.  There is no good way (or any way?) to provide AWS credentials to `rpm-ostree compose`, so we had to fall back to the internal repo.

Reply:
Thanks!